src/app/app.component.html

```diff
@@ -358,6 +358,15 @@
                             </li>
                           </ul>
 
+                          <!-- Response duplication button -->
+                          <button
+                            *ngIf="!(deleteCurrentRouteResponseRequested$ | async)"
```

Comment:
I think you could avoid hiding the button when a deletion has been requested but not confirmed. I don't think it's really necessary. And the user could have misclicked on the trash icon and would have to wait for the timeout in order to duplicate.